include/nil/crypto3/math/polynomial/polynomial_dfs.hpp

```diff
@@ -438,6 +465,30 @@ namespace nil {
                     return result;
                 }
 
+                /**
+                 * Computes the standard polynomial addition, polynomial A -= polynomial B, and stores result in
+                 * polynomial C.
+                 */
+                polynomial_dfs operator-=(const polynomial_dfs& other) {
+                    this->_d = std::max(this->_d, other._d);
+                    if (other.size() > this->size()) {
+                        this->resize(other.size());
+                    }
+                    if (this->size() > other.size()) {
+                        polynomial_dfs tmp(other);
+                        tmp.resize(this->size());
+
+                        std::transform(tmp.begin(), tmp.end(), this->begin(), this->begin(), std::minus<FieldValueType>());
+                        return *this;
+                    }
+                    std::transform(this->begin(), this->end(), other.begin(), this->begin(), std::minus<FieldValueType>());
+                    return *this;
+                }
+
+                polynomial_dfs operator-=(const FieldValueType& c) {
+                    for( auto it = this->begin(); it!=this->end(); it++) *it -= c;
+                    return *this;
+                }
                 /**
                  * Perform the multiplication of two polynomials, polynomial A * polynomial B, and stores result in
```

Comment:
Words don't agree grammatically:

> Perform... and stores

I like the infinitive option more:

```suggestion
                 * Perform the multiplication of two polynomials: polynomial A * polynomial B,
                 * and store the result in polynomial C.
```